building-with-ocean/build-a-marketplace/customising-your-market.md

```diff
@@ -0,0 +1,184 @@
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-21.png)
+
+If you’re interested in doing some further customization, take a look at lines 53 to 64. You can change these properties to alter how the image looks. Feel free to play around with it. We’ve increased the number of layers from 4 to 5.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-22.png)
+
+And now your customized publish page is ready for your customers:
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-20.png)
+
+## Change the Fee Address
+
+At this point, we have made a lot of changes and hopefully you’re happy with the way that your marketplace is looking. Given that you now have your own awesome photo marketplace, it’s about time we talked about monetizing it. Yup, that’s right - you will earn a commission when people buy and sell photos in your marketplace. In Ocean V4, there are a whole host of new fees and customization options that you can use (read more about that here).
+
+When someone sets the pricing for their photos in your marketplace, they are informed that a commission will be sent to the owner of the marketplace. You see that at the moment this fee is set to zero, so you’ll want to increase that. And in order to receive the fees you’ll you need to set the address that you want to recieve these fees in.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-23.png)
+
+This important step is the last thing that we will change in this guide. To set the marketplace fees and address, you’ll need to save them as environmental variables. Create a new file called .env in the root of your repository.
+
+Copy and paste the following into the file:
+
+```
+
+NEXT_PUBLIC_MARKET_FEE_ADDRESS="0x123abc"
+NEXT_PUBLIC_PUBLISHER_MARKET_ORDER_FEE="0.01"
+NEXT_PUBLIC_PUBLISHER_MARKET_POOL_SWAP_FEE="0.01"
+NEXT_PUBLIC_PUBLISHER_MARKET_FIXED_SWAP_FEE="0.01"
+NEXT_PUBLIC_CONSUME_MARKET_ORDER_FEE="0.01"
+NEXT_PUBLIC_CONSUME_MARKET_POOL_SWAP_FEE="0.01"
+NEXT_PUBLIC_CONSUME_MARKET_FIXED_SWAP_FEE="0.01"
+
+```
+
+You need to replace “0x123abc” with your ethereum address (this is where the fees will be sent) and alter the fees to the levels that you intend them to be at. If you change you mind, these fees can always be altered later.
+
+Go to [Fees page](https://docs.oceanprotocol.com/concepts/fees/) to know more details about each type of fee and its relevance.
+
+It is important that the file is saved in the right place at the root of your repository, your file structure should look the same as below.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-24.png)
+
+Now that’s it; you now have a fully functioning photo marketplace that operates over the blockchain. Everytime someone uses it, you will receive revenue.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-25.png)
```

Comment:
In this image, the dynamic pricing schema is used.

Reply:
Yep, I've updated it